packages/twenty-server/src/workspace/workspace-query-runner/jobs/record-position-backfill.job.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+
+import { MessageQueueJob } from 'src/integrations/message-queue/interfaces/message-queue-job.interface';
+
+import { RecordPositionBackfillService } from 'src/workspace/workspace-query-runner/services/record-position-backfill-service';
+
+export type RecordPositionBackfillJobData = {
+  workspaceId: string;
+  objectMetadata: { nameSingular: string; isCustom: boolean };
+  recordId: string;
+};
+
+@Injectable()
+export class RecordPositionBackfillJob
+  implements MessageQueueJob<RecordPositionBackfillJobData>
+{
+  constructor(
+    private readonly recordPositionBackfillService: RecordPositionBackfillService,
+  ) {}
+
+  async handle(data: RecordPositionBackfillJobData): Promise<void> {
+    this.recordPositionBackfillService.backfill(
```

Comment:
missing await?

Reply:
why so? I do not need to return the value anyway so why waiting the result?